core/templates/dev/head/components/answer_group_editor.html

```diff
@@ -77,6 +77,7 @@
     </form>
 
     <div class="oppia-rule-save-cancel-buttons">
+    <p ng-show="editAnswerGroupForm.editFeedbackForm.$invalid && editAnswerGroupForm.editFeedbackForm.newStateName.$dirty" class="help-block oppia-form-error">Please use a non-empty description consisting of alphanumeric characters, spaces and/or hyphens.</p>
```

Comment:
This is the editor for the feedback section ("Oppia tells the learner..."). I don't think you want the error message here.